<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpExtractorPlus</class>
 <widget class="QDialog" name="IpExtractorPlus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input (ips.txt)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browse_input">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browse_output">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_output">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output (result.txt)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_output">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>690</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>471</width>
     <height>511</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
